### PR TITLE
Docker Desktop 4.26.0 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
### Proposed changes

Docker desktop's 4.26.0 release notes

### CODE REVIEW

File: content/desktop/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+{{< release-date date="2023-11-30" >}}
+
+{{< desktop-install all=true version="4.26.0" build_path="/TBD/" >}}
+
+### New
+
+- Organization admins can now control the access to Beta and Experimental features via Docker Desktop's [Features in development](https://docs.docker.com/desktop/settings/mac/#features-in-development).
+- Improved the text and position of the startup options in Settings.
+- Introduced four new version update states in the footer: user call-to-action for update and restart, including download progress.
+- Docker init (Beta) now supports PHP with Apache + Composer.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:38]:
```suggestion
- `docker init` (Beta) now supports PHP with Apache + Composer.
```

File: content/desktop/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+  - Fixed `docker image list` to show real image creation date.
+  - Added support for `-a` flag to `docker pull` (pull all remote repository tags).
+  - Added support for `--group-add` flag to `docker run` (append extra groups).
+  - Adjusted some errors reported by `docker push/pull`.
+- Docker Init:
+  - Improved cross-compilation in Dockerfiles for Golang and Rust.
+  - Improved caching in Dockerfile for ASP.NET Core.
+- You can now switch between different contexts from the UI.
+- Docker Desktop now gives more detailed information about pending updates in the dashboard footer.
+- [Settings Management](hardened-desktop/settings-management/index.md) now lets you turn off Experimental and Beta features for your organisation.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:77]: Can remove this as it is repeated above

File: content/desktop/release-notes.md [hunk 3]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed a bug in Enhanced Container Isolation mode where "docker run --init" was failing.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:78]:
```suggestion
- Fixed a bug in Enhanced Container Isolation mode where `docker run --init` was failing.
```

File: content/desktop/release-notes.md [hunk 4]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed a bug where a notification prompting the user to download a new version of Docker Desktop remained visible after the user started downloading the new version.
+- Added a notification that indicates when Docker Desktop is installing a new version.
+- Fixed a bug where the cursor changed to pointer when the user hovered over a notification that has no call to action.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:81]:
```suggestion
- Fixed a bug where the cursor changed to a pointer when the user hovered over a notification that has no call to action.
```

File: content/desktop/release-notes.md [hunk 5]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+#### For Mac
+
+- Fixed an issue where Rosetta would not work with PHP. Fixes [docker/for-mac#6773](https://github.com/docker/for-mac/issues/6773)  and [docker/for-mac#7037](https://github.com/docker/for-mac/issues/7037).
+- Fixed several issues related to Rosetta not work. Fixed [[docker/for-mac#6973](https://github.com/docker/for-mac/issues/6973), [[docker/for-mac#7009](https://github.com/docker/for-mac/issues/7009), [[docker/for-mac#7068](https://github.com/docker/for-mac/issues/7068) and [[docker/for-mac#7075](https://github.com/docker/for-mac/issues/7075)
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:86]:
```suggestion
- Fixed several issues related to Rosetta not working. Fixed [[docker/for-mac#6973](https://github.com/docker/for-mac/issues/6973), [[docker/for-mac#7009](https://github.com/docker/for-mac/issues/7009), [[docker/for-mac#7068](https://github.com/docker/for-mac/issues/7068) and [[docker/for-mac#7075](https://github.com/docker/for-mac/issues/7075)
```

File: content/desktop/release-notes.md [hunk 6]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Improved performance of NodeJS under Rosetta.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:87]:
```suggestion
- Improved the performance of NodeJS under Rosetta.
```

File: content/desktop/release-notes.md [hunk 7]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed the `Unable to open /proc/self/exe` Rosetta errors.
+- Fixed a bug were the setting `Start Docker Desktop when you log` would not work. Fixes [docker/for-mac#7052](https://github.com/docker/for-mac/issues/7052).
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:89]:
```suggestion
- Fixed a bug were the setting **Start Docker Desktop when you sign in** would not work. Fixes [docker/for-mac#7052](https://github.com/docker/for-mac/issues/7052).
```

File: content/desktop/release-notes.md [hunk 8]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- You can now use enable the use of Kernel networking path for UDP through the UI. Fixes [docker/for-mac#7008](https://github.com/docker/for-mac/issues/7008).
+- Fixed a regression where the `uninstall` CLI tool was missing.
+
+#### For Windows
+
+- Added support for WSL mirrored mode networking (requires WSL `v2.0.4` and up).
+- Added missing signatures on DLL and VBS files
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:96]:
```suggestion
- Added missing signatures on DLL and VBS files.
```

File: content/desktop/release-notes.md [hunk 9]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed a bug where a notification prompting the user to download a new version of Docker Desktop remained visible after the user started downloading the new version.
+- Added a notification that indicates when Docker Desktop is installing a new version.
+- Fixed a bug where the cursor changed to pointer when the user hovered over a notification that has no call to action.
+
+#### For Mac
+
+- Fixed an issue where Rosetta would not work with PHP. Fixes [docker/for-mac#6773](https://github.com/docker/for-mac/issues/6773)  and [docker/for-mac#7037](https://github.com/docker/for-mac/issues/7037).
+- Fixed several issues related to Rosetta not work. Fixed [[docker/for-mac#6973](https://github.com/docker/for-mac/issues/6973), [[docker/for-mac#7009](https://github.com/docker/for-mac/issues/7009), [[docker/for-mac#7068](https://github.com/docker/for-mac/issues/7068) and [[docker/for-mac#7075](https://github.com/docker/for-mac/issues/7075)
+- Improved performance of NodeJS under Rosetta.
+- Fixed the `Unable to open /proc/self/exe` Rosetta errors.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:88]:
```suggestion
- Fixed the **Unable to open /proc/self/exe** Rosetta errors.
```

File: content/desktop/release-notes.md [hunk 10]
```diff
@@ -24,6 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed a bug were the setting `Start Docker Desktop when you log` would not work. Fixes [docker/for-mac#7052](https://github.com/docker/for-mac/issues/7052).
+- You can now use enable the use of Kernel networking path for UDP through the UI. Fixes [docker/for-mac#7008](https://github.com/docker/for-mac/issues/7008).
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:90]:
```suggestion
- You can now enable the use of Kernel networking path for UDP through the UI. Fixes [docker/for-mac#7008](https://github.com/docker/for-mac/issues/7008).
```

File: content/desktop/release-notes.md [hunk 11]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+  - Fixed `docker tag` not preserving internal metadata.
+  - Fixed `docker commit` when the daemon configured with `--userns-remap`.
+  - Fixed `docker image list` to show real image creation date.
+  - Added support for `-a` flag to `docker pull` (pull all remote repository tags).
+  - Added support for `--group-add` flag to `docker run` (append extra groups).
+  - Adjusted some errors reported by `docker push/pull`.
+- Docker Init:
+  - Improved cross-compilation in Dockerfiles for Golang and Rust.
+  - Improved caching in Dockerfile for ASP.NET Core.
+- You can now switch between different contexts from the UI.
```
Comment (MihaelaStoica) [inline content/desktop/release-notes.md:75]: I am not sure this should be included, as it is behind a feature flag.
cc @stanislavHamara @djs55

File: content/desktop/release-notes.md [hunk 12]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Docker Desktop now gives more detailed information about pending updates in the dashboard footer.
+- Fixed a bug in Enhanced Container Isolation mode where `docker run --init` was failing.
+- Fixed a bug where a notification prompting the user to download a new version of Docker Desktop remained visible after the user started downloading the new version.
+- Added a notification that indicates when Docker Desktop is installing a new version.
+- Fixed a bug where the cursor changed to a pointer when the user hovered over a notification that has no call to action.
+
```
Comment (trungutt) [inline content/desktop/release-notes.md:81]: @rebecapereira-docker we should add a note here for the new Appbar for Docker Desktop

File: content/desktop/release-notes.md [hunk 13]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+{{< desktop-install all=true version="4.26.0" build_path="/TBD/" >}}
+
+### New
+
+- Administrators can now control the access to beta and experimental features in the **Features in development** tab with [Settings Management](hardened-desktop/settings-management/configure.md).
+- Improved the text and position of the startup options in the **General** settings tab.
+- Introduced four new version update states in the footer.
+- `docker init` (Beta) now supports PHP with Apache + Composer.
+- The [**Builds** view](use-desktop/build.md) is now GA. You can now inspect builds, troubleshoot errors, and optimize build speed.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:39]:
```suggestion
- The [**Builds** view](use-desktop/builds.md) is now GA. You can now inspect builds, troubleshoot errors, and optimize build speed.
```

File: content/desktop/release-notes.md [hunk 14]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 
+## 4.26.0
+
+{{< release-date date="2023-11-30" >}}
+
+{{< desktop-install all=true version="4.26.0" build_path="/TBD/" >}}
+
+### New
+
+- Administrators can now control the access to beta and experimental features in the **Features in development** tab with [Settings Management](hardened-desktop/settings-management/configure.md).
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:35]:
```suggestion
- Administrators can now control access to beta and experimental features in the **Features in development** tab with [Settings Management](hardened-desktop/settings-management/configure.md).
```

File: content/desktop/release-notes.md [hunk 15]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Improved the text and position of the startup options in the **General** settings tab.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:36]: Think we can actually move this to bug fixes and enhancements section

File: content/desktop/release-notes.md (hunk 11 above)
Comment (lorenrh) [reply content/desktop/release-notes.md:75]: removed it ! :D

File: content/desktop/release-notes.md [hunk 16]
```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- [Docker Engine v24.0.7](https://docs.docker.com/engine/release-notes/24.0/#2407)
+- [Containerd v1.6.25](https://github.com/containerd/containerd/releases/tag/v1.6.25)
+- [runc v1.1.10](https://github.com/opencontainers/runc/releases/tag/v1.1.10)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- You can now provide feedback from the commandline by using `docker feedback`.
+- Improved the text and position of the startup options in the **General** settings tab.
```
Comment (mark-dr) [inline content/desktop/release-notes.md:61]: Proposing a line for for design system changes (wording discussed and agreed within frontend platform team):

```suggestion
- Improved the text and position of the startup options in the **General** settings tab.
- Redesigned the dashboard's header bar, added links to other Docker resources, improved display of account information.
```

**Review by mark-dr — CHANGES_REQUESTED** [review]

File: content/desktop/release-notes.md [hunk 17]
```diff
@@ -24,6 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.26.0
+
+{{< release-date date="2023-11-30" >}}
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:29]:
```suggestion
{{< release-date date="2023-12-04" >}}
```

**Review by aevesdocker — APPROVED** [review]
Merci @lorenrh ⭐

**Pull request merged**